src/resultset/formatters/resultset_replynop.h

```diff
@@ -9,18 +9,16 @@
 // Formatter for compact (client-parsed) replies
 void ResultSet_EmitNOPHeader
 (
-    RedisModuleCtx *ctx,
-    bolt_client_t *bolt_client,
-    const char **columns,
-    uint *col_rec_map
+    ResultSet *set
 );
 
 void ResultSet_EmitNOPRow
 (
-    RedisModuleCtx *ctx,
-    bolt_client_t *bolt_client,
-    GraphContext *gc,
-    SIValue **row,
-    uint numcols
+    ResultSet *set,
+    SIValue **row
 );
 
+void ResultSet_EmitNOPStats
```

Comment:
As with the other formatters header files, please make sure each function in the header file is documented.